osgi/karaf/features/src/main/resources/org.apache.karaf.features.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<featuresProcessing xmlns="http://karaf.apache.org/xmlns/features-processing/v1.0.0" xmlns:f="http://karaf.apache.org/xmlns/features/v1.6.0">
+    <blacklistedBundles>
+        <bundle>mvn:org.ops4j.pax.web/pax-web-undertow/7.3.13</bundle>
```

Comment:
The issue with servlet dependencies (see please http://karaf.922171.n3.nabble.com/karaf-4-3-0-unable-to-install-hawtio-core-with-pax-http-undertow-td4059219.html).

```
[ERROR]                                                                                                                               
[ERROR] Chain 1:                                                                                                                      
[ERROR]   org.ops4j.pax.web.pax-web-undertow [org.ops4j.pax.web.pax-web-undertow/7.3.13]                                              
[ERROR]     import: (&(osgi.wiring.package=javax.servlet)(version>=3.1.0)(!(version>=5.0.0)))                                         
[ERROR]      |                                                                                                                        
[ERROR]     export: osgi.wiring.package: javax.servlet                                                                                
[ERROR]   javax.servlet-api [javax.servlet-api/3.1.0]                                                                                 
[ERROR]                                                                                                                               
[ERROR] Chain 2:                                                                                                                      
[ERROR]   org.ops4j.pax.web.pax-web-undertow [org.ops4j.pax.web.pax-web-undertow/7.3.13]                                              
[ERROR]     import: (&(osgi.wiring.package=io.undertow.servlet)(version>=2.0.0)(!(version>=2.1.0)))                                   
[ERROR]      |                                                                                                                        
[ERROR]     export: osgi.wiring.package=io.undertow.servlet; uses:=javax.servlet                                                      
[ERROR]   io.undertow.servlet [io.undertow.servlet/2.0.33.Final]                                                                      
[ERROR]     import: (&(osgi.wiring.package=javax.servlet)(version>=4.0.0)(!(version>=5.0.0)))                                         
[ERROR]      |                                                                                                                        
[ERROR]     export: osgi.wiring.package: javax.servlet                                                                                
[ERROR]   javax.servlet-api [javax.servlet-api/4.0.0]                                                                                 
[ERROR] Repositories: {                                                                                                               
[ERROR]         file:C:\Users\retab\Workspaces\apache-cxf-git\cxf-git\osgi\karaf\features\target/classes/features.xml                 
[ERROR]         mvn:org.apache.karaf.features/framework/4.3.1/xml/features                                                            
[ERROR]         mvn:org.apache.karaf.features/spring/4.3.1/xml/features                                                               
[ERROR]         mvn:org.apache.karaf.features/standard/4.3.1/xml/features                                                             
[ERROR]         mvn:org.ops4j.pax.cdi/pax-cdi-features/1.1.4/xml/features                                                             
[ERROR]         mvn:org.ops4j.pax.web/pax-web-features/7.3.13/xml/features                                                            
[ERROR] }                                                                                                                             
```
Looking for the proper way to solve that but blacklisting `pax-web-undertow` for now (cannot blacklist `javax.servlet/3.1.0` , breaks whole bunch of features at the moment)